src/resmokeconfig/genny_sharded.yml

```diff
@@ -12,7 +12,7 @@ executor:
     - "12345"
     - "--success"
     - "--out"
-    - "build/test_gennylib_with_server.junit.xml"
+    - "build/cast_core_test.junit.xml"
```

Comment:
so now resmoke only runs `cast_core_test` so any tests within `gennylib/test` that require a mongo db won't be run by resmoke. I don't see a good way around this but open to ideas - @guoyr is it possible to specify multiple binaries or something?

I think the main value of running via resmoke is for Actor-authors who probably don't care much about running genny's own internal tests so I'm not super concerned but if it's easy to fix I'd like to do so.

Reply:
Not at the moment; it's a known limitation. I'll have a CR up to add support. Should be easy fix.

Reply:
Filed https://jira.mongodb.org/browse/SERVER-39125